commands/install_license

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+help() {
+  cat <<EOF
+Usage: $(basename "$0") [--license <license-key>]
+Ex: c8y oee \install_license --license AbcD4E567
```

Comment:
now that I think about it, maybe a second option using a file would be nice as well. or at least an example how the user could read the content of the file and pipe it into the "--licence" parameter; e.g. `c8y oee \install_license --license | cat myLicense.xml`